src/cc/bcc_elf.c

```diff
@@ -287,15 +290,22 @@ static int list_in_scn(Elf *e, Elf_Scn *section, size_t stridx, size_t symsize,
 #endif
 #endif
 
-      if (callback(name, sym.st_value, sym.st_size, payload) < 0)
+      int ret;
+      if (callback_lazy)
```

Comment:
Can we put this in `bcc_symbol_option` so that we could reliably select which option to use, avoid people get confused and specifying both etc.?